mwdb/web/src/commons/ui/ErrorBoundary.jsx

```diff
@@ -58,6 +90,7 @@ function CriticalError(props) {
     );
 }
 
+//TODO: rewrite to function component
```

Comment:
I'm not sure if it's possible to make error boundary without using class components

Ref: https://reactjs.org/docs/error-boundaries.html

Reply:
ok, i removed this comment